packages/dcos-integration-test/extra/test_3dt.py

```diff
@@ -496,6 +498,22 @@ def test_3dt_bundle_download_and_extract(dcos_api_session):
     """
     test bundle download and validate zip file
     """
+    _download_bundle(dcos_api_session)
+
+
+def test_3dt_bundle_download_and_extract_from_another_master(dcos_api_session):
+    """
+    test bundle download and validate zip file
+    """
+    if len(dcos_api_session.masters) < 3:
+        pytest.skip('Test requires at least 3 master nodes')
+
+    _download_bundle(dcos_api_session, master_index=1)
+
+
+def _download_bundle(dcos_api_session, master_index=0):
```

Comment:
Please make master_index a regular required arg. It seems a little confusing as a default. And perhaps call this function '_download_bundle_from_master'.